.babelrc

```diff
@@ -1,7 +1,4 @@
 {
-  "plugins": [
```

Comment:
Nit: Since it is not picked up here, move it into the env objects below instead of putting it in the command lines, to keep config in one place?